packages/experiments/src/components/FolderCover/FolderCover.scss

```diff
@@ -53,6 +53,10 @@
   .hideContent & {
     opacity: 0;
   }
+
+  .isFluent & {
```

Comment:
You can now change the `&.isFluent` rule in `.metadata` to `.isFluent & {` and then stop passing `isFluent` to the `.metadata` element directly.